src/main/java/org/hpccsystems/jdbcdriver/HPCCConnection.java

```diff
@@ -118,8 +122,13 @@ public HPCCConnection(Properties props)
                     this.wsEclPort = eclWatchURL.getPort();
 
                 hpccPlatform = Platform.get(eclWatchURL.getProtocol(), eclWatchURL.getHost(), wsEclPort, userName, props.getProperty("password", ""));
+                if (hpccPlatform.isDisabled())
+                {
+                    HPCCJDBCUtils.traceoutln(Level.SEVERE, "The HPCC platform is disabled for host: " + eclWatchURL.getProtocol() + "://" + eclWatchURL.getHost() + ":" + wsEclPort);
```

Comment:
It seems we should bail if isDisabled(),
Also something like this:
addWarning(new SQLWarning("The HPCC platform is disabled for host: " + eclWatchURL.getProtocol() + "://" + eclWatchURL.getHost() + ":" + wsEclPort));